numpy/random/mtrand/mtrand.pyx

```diff
@@ -4997,6 +4997,9 @@ cdef class RandomState:
 
         Modify a sequence in-place by shuffling its contents.
 
+        This function only shuffles the array along the first index of a
```

Comment:
I moved this to the main function description because it is very important and I did not notice it in the example the fist time I read the docs.